src/textures/Texture.js

```diff
@@ -97,7 +112,7 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 
 		this.name = source.name;
 
-		this.image = source.image;
+		this.textureImage = source.textureImage;
 		this.mipmaps = source.mipmaps.slice( 0 );
```

Comment:
Rebuild a repo please, this PR seems wrong.
![](https://habrastorage.org/webt/95/ma/uz/95mauzbwyhcilln3lm8ziesqziu.png)

Reply:
Lack in deterministic Math.Random().

Reply:
The color of the point light is random. Could this be a problem?

https://github.com/mrdoob/three.js/blob/2c73923c3ee412e048f544603dce04c73ebec44b/examples/webgl_loader_ttf.html#L74